floris/simulation/wake_velocity/cumulative_gauss_curl.py

```diff
@@ -175,21 +175,24 @@ def function(
         a2 = 2 ** (4 / n - 2)
 
         # based on Blondel model, modified to include cumulative effects
-        C = a1 - np.sqrt(
-            a2
-            - (
-                (n * turbine_Ct[:,:,ii:ii+1])
-                * cosd(turbine_yaw)
-                / (
-                    16.0
-                    * gamma(2 / n)
-                    * np.sign(sigma_n)
-                    * (np.abs(sigma_n) ** (4 / n))
-                    * (1 - sum_lbda) ** 2
-                )
+        tmp = a2 - (
```

Comment:
Should this fix go in a separate Pull Request? Is this generic enough to be merged back into `develop` or is this just a temporary fix for something?